CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
```

Comment:
For what it's worth, merging changes like this is the "normal" approach we take to applying updates to gem's here. Bumping with a change should mostly be the exception, though I think that's slipped a bit in recent years.

Reply:
Ah good to know 👍